vnode_controller/vnode_controller.go

```
@@ -149,7 +155,7 @@
 
 	go func() {
 		// wait for all tunnel to be ready
-		utils.CheckAndFinallyCall(context.Background(), func() (bool, error) {
+		utils.CheckAndFinallyCall(context.Background(), func(ctx context.Context) (bool, error) {
```

golangci-lint (1.62.2)

Error return value of `utils.CheckAndFinallyCall` is not checked

(errcheck)